types/react/index.d.ts

```diff
@@ -123,6 +124,13 @@ declare namespace React {
         type: keyof ReactSVG;
     }
 
+    interface ReactPortal {
+        key: Key | null;
+        containerInfo: any;
+        children: ReactNode;
+        implementation: any;
```

Comment:
I appreciate that you've gone and found the actual implementation and described it here, but since none of these properties are [documented](https://reactjs.org/docs/portals.html) I'm going to suggest that we leave some of them out. The `any`-typed properties in particular I think we should drop.